projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -290,6 +290,23 @@ public static function load_features() {
 		require_once __DIR__ . '/features/wpcom-widgets/wpcom-widgets.php';
 		require_once __DIR__ . '/features/wpcom-wpadmin-page-view/wpcom-wpadmin-page-view.php';
 
+		/**
```

Comment:
I think all the inline logic here is out of place and does not belong here. The function comment explicitly says:

```php
	/**
	 * Load features that don't need any special loading considerations.
	 */
	public static function load_features() {
```

Maybe create another function like `load_newspack_blocks()`?

Reply:
Created by https://github.com/Automattic/jetpack/pull/44906/commits/8e01271005854028a7b82d15aedf4d115e29de02.